README.md

```diff
@@ -1,7 +1,32 @@
-# IPLD Roadmap
-
 IPLD Project Roadmap
```

Comment:
Should we remove this line completely, it looks strange to me having some text before the headline.

Reply:
The way GitHub shows the README text on the main page ends up displaying this rather nicely, it just doesn't look great in the raw markdown version :(